poetry.lock

```diff
@@ -21,7 +21,7 @@ typing_extensions = ">=3.7.2"
 name = "alabaster"
 version = "0.7.12"
 description = "A configurable sidebar-enabled Sphinx theme"
-category = "main"
+category = "dev"
```

Comment:
I would omit changes to this during development -- it just adds a bunch of noise to the commits.

Reply:
I'm not trying to change it!!